doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,197 @@ To send bare Lua code for execution, use ``Eval``:
+        -   No
+        -   No
+
+    *   -   `Varbinary <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes (with in-built language tools)
+        -   Yes (with in-built language tools)
+        -   No (decodes to string by default, see `#6 <https://github.com/FZambia/tarantool/issues/6>`_)
+
+    *   -   `UUID <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes
+        -   No
+        -   No
+
+    *   -   Decimal support
+        -   No (`#96 <https://github.com/tarantool/go-tarantool/issues/96>`_)
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`_
+            support
+        -   No
+        -   No
+        -   No
+
+    *   -   `Datetime <https://github.com/tarantool/tarantool/discussions/6244>`_ support
+        -   No (`#118 <https://github.com/tarantool/go-tarantool/issues/118>`_)
+        -   No
+        -   No
+
+    *   -   `box.session.push() responses <https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_session/push/>`_
+        -   No (`#67 <https://github.com/tarantool/go-tarantool/issues/67>`_)
+        -   No (`#21 <https://github.com/viciious/go-tarantool/issues/21>`_)
+        -   Yes
+
+    *   -   `Session settings <https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_session_settings/>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `Graceful shutdown <https://github.com/tarantool/tarantool/issues/5924>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `IPROTO_ID (feature discovering) <https://github.com/tarantool/tarantool/issues/6253>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   `tarantool/crud <https://github.com/tarantool/crud>`_ support
+        -   No
+        -   No
+        -   No
+
+    *   -   Connection pool
+        -   Yes (round-robin failover; no balancing; master discovering planned in `#113 <https://github.com/tarantool/go-tarantool/issues/113>`_)
+        -   No
+        -   No
+
+    *   -   Implicit reconnecting strategy
+        -   Yes (see comments in `#129 <https://github.com/tarantool/go-tarantool/issues/129>`_)
+        -   No (handle reconnects explicitly, refer to `#11 <https://github.com/viciious/go-tarantool/issues/11>`_)
+        -   Yes (see comments in `#7 <https://github.com/FZambia/tarantool/issues/7>`_)
+
+    *   -   Support retrying
+        -   No
+        -   No
+        -   No
+
+    *   -   `Watchers <https://github.com/tarantool/tarantool/pull/6510>`_
+        -   No
+        -   No
+        -   No
+
+    *   -   Language features
+        -   No  (`#48 <https://github.com/tarantool/go-tarantool/issues/48>`_)
+        -   context
+        -   context
+
+    *   -   Other features
+        -   Queue API
+        -   Can mimic a Tarantool instance (also as replica)
+        -   No
+
+Last update: December 2021
```

Comment:
I would place the date of last update before the table. Because if the table is up-to-date, then there will be more trust to it.

(Feel free to ignore my comment).

Reply:
Well, that's fair (and consistent with Python table). Moved the date